config/git2consul.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "1.0",
+    "repos" : [{
+        "name" : "config/store",
+        "url" : "https://github.com/ahmadiq/storeconfig.git",
```

Comment:
also each config file shld be inside each microservice source code e.g. src/main/resources/config